scripts/jenkins/jobs-ibs/cloud-mkcloud7-gating.yaml

```diff
@@ -0,0 +1,56 @@
+- job:
+    name: 'cloud-mkcloud7-gating'
+    project-type: multijob
+    node: cloud-trigger
+    disbaled: true
+
+    triggers:
+      - timed: 'H 19 */2 * *'
```

Comment:
same time that cloud 6, makes sense to change it to avoid over-usage of mkcloud servers?